src/index.js

```diff
@@ -14,11 +15,13 @@ ReactDOM.render(
   <React.StrictMode>
     <BrowserRouter>
       <CartProvider>
-        <WishlistProvider>
-          <FilterProvider>
-            <App />
-          </FilterProvider>
-        </WishlistProvider>
+        <AuthProvider>
+          <WishlistProvider>
+            <FilterProvider>
+              <App />
```

Comment:
for now its fine, but in future try using one or at most 2 contexts, that will look better

Reply:
Sure, I will


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Tags:** Efficient